ShoppingList.ConsoleApp/Models/ShoppingItems/ShoppingItem.cs

```diff
@@ -0,0 +1,15 @@
+﻿// ------------------------------------------------
+// Copyright (c) MumsWhoCode. All rights reserved.
+// ------------------------------------------------
+
+namespace ShoppingList.ConsoleApp.Models.ShoppingItems
+{
+    class ShoppingItem
+    {
+        public int Id { get; set; }
+        public string Text { get; set; }
```

Comment:
Name would be a more appropriate variable name.
Also just have 3 maximum. We will add more attributed later. So Id + 2 more.